cmd/node/config/economics.toml

```diff
@@ -3,13 +3,13 @@
     TotalSupply = "20000000000000000000000000000" #20BILERD
     MinimumInflation = 0.0
     MaximumInflation = 0.075 #fraction of value 1 - 7.5%
+    Denomination = 18
```

Comment:
you can add a comment to say what this parameter is

Reply:
done